corehq/tests/noseplugins/dividedwerun.py

```diff
@@ -84,7 +84,7 @@ def skip():
                         skip.__name__ = "*"
                     skip.__module__ = name
             else:
-                desc = test.test
+                desc = test.test.descriptor
```

Comment:
I'm fine with always using `test.test.descriptor` as long as it always exists and it has a descriptive (i.e., not generic) `__name__` for the test